api/setup.py

```diff
@@ -9,6 +9,7 @@
 import sys
 
 import itertools
+from setuptools import find_packages
```

Comment:
Shouldn't this import for `find_packages` be further down? We want to account for when people don't have `setuptools`